keter-backend/src/main/java/ru/avplatonov/keter/backend/Application.java

```diff
@@ -0,0 +1,60 @@
+package ru.avplatonov.keter.backend;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import ru.avplatonov.keter.backend.db.GraphsDB;
+import ru.avplatonov.keter.backend.db.NodesDB;
+
+import java.util.Arrays;
+
+@SpringBootApplication
+public class Application {
+
+    public static ApplicationContext context;
+    public static GraphsDB graphsDB;
+    public static NodesDB nodesDB;
+
+	public static void main(String[] args) {
+        context = new AnnotationConfigApplicationContext(GraphsDB.class, NodesDB.class);
+        graphsDB = context.getBean(GraphsDB.class);
```

Comment:
Use dep. injections through annontations
Don't use static fields and getBean explicitly